pulpcore.fc

```diff
@@ -1,3 +1,13 @@
+# label for both nginx & apache
+/etc/pulp/certs(/.*)?			gen_context(system_u:object_r:httpd_config_t,s0)
+/etc/pulp/settings.py			gen_context(system_u:object_r:pulpcore_etc_t,s0)
+/etc/pulp/token_private_key.pem			gen_context(system_u:object_r:pulpcore_etc_t,s0)
+/etc/pulp/token_public_key.pem			gen_context(system_u:object_r:pulpcore_etc_t,s0)
+# The installer prior to 2020-08-20 used these 2 token paths. Clean up once dev
+# instance is deleted.
+/etc/pulp/private_key.pem			gen_context(system_u:object_r:pulpcore_etc_t,s0)
+/etc/pulp/public_key.pem			gen_context(system_u:object_r:pulpcore_etc_t,s0)
```

Comment:
I'd expect this to conflict with the pulp2 policy:
https://github.com/pulp/pulp/blob/2-master/server/selinux/server/pulp-server.fc#L2

Reply:
nah, seems to work fine:

```
/etc/pki/pulp(/.*)?                                all files          system_u:object_r:pulp_cert_t:s0 
/etc/pulp(/.*)?                                    all files          system_u:object_r:httpd_sys_content_t:s0 
/etc/pulp/certs(/.*)?                              all files          system_u:object_r:httpd_config_t:s0 
/etc/pulp/private_key.pem                          all files          system_u:object_r:pulpcore_etc_t:s0 
/etc/pulp/public_key.pem                           all files          system_u:object_r:pulpcore_etc_t:s0 
/etc/pulp/settings.py                              all files          system_u:object_r:pulpcore_etc_t:s0 
/etc/pulp/token_private_key.pem                    all files          system_u:object_r:pulpcore_etc_t:s0 
/etc/pulp/token_public_key.pem                     all files          system_u:object_r:pulpcore_etc_t:s0 
```

Reply:
not sure about the `httpd_sys_content_t` vs `httpd_config_t`, but I have no idea what these do

Reply:
After applying this on my Katello install, `/etc/pulp/settings.py` was (obviously) relabeled, but things seem to work fine still.